activerecord/CHANGELOG.md

```diff
@@ -32,6 +32,24 @@
 
     *Kevin Newton*
 
+*   Previously, some `configuration_hash`'s keys for `ActiveRecord::DatabaseConfigurations::UrlConfig`
```

Comment:
Sorry, on the move this week. 
I think the title could be a bit more succinct. What do you think about:
```suggestion
*   Allow configuring Postgres password through the socket URL.
```

Reply:
Yes, this is more clearer as to what this PR achieves. I've made changes to the CHANGELOG.md. Thanks for the help.